src/Generator.hh

```diff
@@ -25,8 +25,6 @@ namespace google {
 namespace protobuf {
 namespace compiler {
 namespace cpp {
-class GeneratorContext;
```

Comment:
declaration 'GeneratorContext' is never referenced, but a declaration with the same name found in another namespace 'proto2::compiler'